Mapping/SolARPipeline_Mapping_Mono/tests/SolARPipelineTest_Mapping_Mono/SolARPipelineTestMappingMono_main.cpp

```diff
@@ -115,12 +115,12 @@ int main(int argc, char ** argv)
 			}
 		}		
 		
-		std::vector<SRef<CloudPoint>> pointCloud;
-		std::vector<Transform3Df> keyframePoses;
-		if (gMappingPipeline->getDataForVisualization(pointCloud, keyframePoses) == FrameworkReturnCode::_SUCCESS) {
-			LOG_INFO("Number of cloud points: {}", pointCloud.size());
-			LOG_INFO("Number of keyframes: {}", keyframePoses.size());
-			while (g3DViewer->display(pointCloud, keyframePoses[keyframePoses.size() - 1], keyframePoses) == FrameworkReturnCode::_SUCCESS);
+		while (true) {
+			std::vector<SRef<CloudPoint>> pointCloud;
+			std::vector<Transform3Df> keyframePoses;
+			if ((gMappingPipeline->getDataForVisualization(pointCloud, keyframePoses) != FrameworkReturnCode::_SUCCESS) ||
+				(g3DViewer->display(pointCloud, keyframePoses[keyframePoses.size() - 1], keyframePoses) == FrameworkReturnCode::_STOP))
+				break;
 		}
 		gMappingPipeline->stop();
```

Comment:
Could be nice to keep the LOG_INFO() ?
Second while is here to continue rendering in 3D view when the video view has been ended, but we could maybe call getDataForVisualization() one last time before stop() to log final result  ? (# of points, # of keyframes...)
(same for Multi)